interpreter/cling/lib/Interpreter/DynamicLibraryManagerSymbol.cpp

```diff
@@ -352,6 +352,7 @@ std::string cached_realpath(llvm::StringRef path, llvm::StringRef base_path = ""
       size_t s = result.rfind(sep);
       if (s != llvm::StringRef::npos) result.resize(s);
       if (result.empty()) result = sep;
```

Comment:
Can you break the statements on the next line?

Reply:
I'm not touching that line...? Happy to see a subsequent PR that reformats the file.

Either way, I have moved that commit out of this PR as we don't seem to converge quickly enough with the realpath test; the commit is now in https://github.com/root-project/root/pull/9951

Reply:
You touched it in #9911. I don't think we need more reformatting than the proposed one - it'd just bloat the history...